src/taco-cli/cli/run.ts

```diff
@@ -186,7 +190,7 @@ class Run extends commands.TacoCommandBase {
             name: "local",
             run: Run.local,
             canHandleArgs(commandData: commands.ICommandData): boolean {
-                return !!commandData.options["local"];
+                return !!commandData.options["local"] || !!commandData.options["livereload"] || !!commandData.options["devicesync"];
```

Comment:
If `--local` or `--remote` are specified then we only do local or remote targets, but if neither is specified then `taco run ios windows` will run both ios and windows, and will decide whether to build them remotely or not based on whether a remote has been configured. This isn't the right place to check that condition, since it is a per-platform value and isn't yet determined.

A better place to check would be runRemotePlatform which is where we try and actually run remotely; you could error out or ignore the options if they are specified there.